app/lib/services/background_service.dart

```diff
@@ -1,6 +1,11 @@
 import 'package:background_fetch/background_fetch.dart';
 import 'package:threebotlogin/services/nodes_check_service.dart';
 import 'notification_service.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:threebotlogin/helpers/logger.dart';
+
+const String _nodeStatusNotificationEnabledKey =
```

Comment:
- The constant is also defined in `background_service.dart`, consider keeping it in a shared place.